examples/data-block-examples/sentinel2-l2-a-geotiff-example.ipynb

```diff
@@ -0,0 +1,302 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "defined-instrument",
+   "metadata": {},
+   "source": [
+    "# Sentinel-2 L2A Analytic (GeoTIFF)\n",
+    "This notebook provides details on how to access Sentinel-2 data (L2A pre-processing or Bottom Of Atmosphere (BOA) reflectance images). \n",
+    "Sentinel-2 Level-2A products have been available over Europe since March 2018, and later extended gloablly as of December 2018."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "static-riding",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import SDK\n",
+    "import up42"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "scientific-haiti",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2021-03-19 16:45:59,477 - Authentication with UP42 successful!\n",
+      "2021-03-19 16:46:00,349 - Initialized Project(name: S2l2a-Analytics-data-example, project_id: 1f2bad99-daad-4a60-9192-5169263caff6, description: , createdAt: 2021-03-18T14:39:08.255254Z)\n"
```

Comment:
Be sure to change analytic**s** to **analytic**. I made the same mistake when I created the block, but markus called to my attention that It doesn't mean the same thing. Analytics would mean that It has already been processed or analysed